docs/core/install/linux-scripted-manual.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Manually install .NET on Linux - .NET
+description: Demonstrates how to install the .NET SDK and the .NET Runtime without a package manager on Linux. Use the install script or manually extract the binaries.
+author: adegeo
+ms.author: adegeo
+ms.date: 12/16/2020
```

Comment:
Commenting here, but it applies to all the articles you've changed. I'd update the date to today:

```suggestion
ms.date: 1/06/2021
```